build_tools/waf_dynamo.py

```diff
@@ -1338,7 +1346,8 @@ def run_tests(ctx, valgrind = False, configfile = None):
 
             cmd = launch_pattern % (program, configfile if configfile else '')
             if 'web' in ctx.env.PLATFORM: # should be moved to TEST_LAUNCH_ARGS
-                cmd = '%s %s' % (ctx.env['NODEJS'], cmd)
+                cmd = '%s %s' % (ctx.env['NODEJS'][0], cmd)
```

Comment:
The "find_program" function returns a list, and we check if the variable has been found earlier in the code